sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,29 @@
+# Kyma Installer with embedded kyma charts and tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Abstract
+
+Current installation process involves downloading kyma charts from cloud storage. Such solution allows to release kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of kyma components using single installer binary. It seems to be a nice feature, but in the reality introduces problems:
+- Build/Release process is more complex because it has to publish kyma charts in cloud storage.
+- It is possible to run installer with kyma charts version not supported (not tested with the installer in given version).
+- Upgrade process is more complex (usually installer has to be upgraded and then kyma release URL has to be updated)
```

Comment:
`The upgrade process is more complex. Usually, the installer upgrade must precede the Kyma release URL update.`